eslint.config.mjs

```diff
@@ -1,6 +1,62 @@
 import baseConfig from '@hono/eslint-config'
 import { defineConfig, globalIgnores } from 'eslint/config'
 
-export default defineConfig(globalIgnores(['.yarn', '**/dist']), {
+export default defineConfig(globalIgnores(['.yarn', '**/coverage', '**/dist']), {
   extends: baseConfig,
+
+  languageOptions: {
+    parserOptions: {
+      projectService: true,
+      tsconfigRootDir: import.meta.dirname,
+    },
+  },
+
+  linterOptions: {
+    reportUnusedDisableDirectives: 'error',
+    reportUnusedInlineConfigs: 'error',
+  },
```

Comment:
Additional [linter options](https://eslint.org/docs/latest/use/configure/configuration-files#configuring-linter-options)

`reportUnusedDisableDirectives` defaults to `'warn'` and `reportUnusedInlineConfigs` defaults to `'off'`